app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/_menu.less

```
@@ -266,6 +266,22 @@
             .ie11 & {
                 height: 100%;
             }
+           & > ul[role="menu"] {
+                max-width:  ~'calc(@{menu__width} - 100vw)';
+                width: 100vw;
+                display: -webkit-inline-flex;
+                display: -ms-inline-flexbox;
+                display: inline-flex;
```

I guess It's better to use the corresponding mixin:
```suggestion
                .lib-vendor-prefix-display(inline-flex);
```